Frontend programare examen Student View/Conectarea la DB

Alex-Adochitei opened the pull request.

Alex-Adochitei committed:
Frontend programare examen Student View/Conectarea la DB

rZn24h commented on ExamScheduler.Server/ExamScheduler.Server.http:
```diff
@@ -1,6 +1,11 @@
 GET {{ExamScheduler.Server_HostAddress}}/weatherforecast/
 Accept: application/json
 
 ###
```
E foarte bine facut!

rZn24h commented on examscheduler.client/src/App.jsx:
```diff
@@ -1,51 +1,169 @@
+        : <table className="table table-striped">
             <thead>
                 <tr>
-                    <th>Date</th>
```
Trebuie adaugate mai multe campuri!

Alex-Adochitei committed:
Rezolvare problema Failed to fetch in swagger

Alex-Adochitei committed:
Conectare Frontend-Backend

Alex-Adochitei committed:
Update

Alex-Adochitei committed:
Update - Adaugare/Afisare Schedule Request

Alex-Adochitei committed:
Update la Frontend

Alex-Adochitei committed:
StudentGroupLider View v1.0

Am adaugat noi liste care arata examenele care se afla in asteptare, care sunt acceptate si care sunt refuzate. Iar la calendar am adaugat posibilitatea de a vedea zilele in care sunt examene si detalii despre acestea. Am conctat Backend-ul cu Frontend-ul.
